### ISSUE TITLE
Upgrading from grav v1.0.10

### ISSUE THREAD

**Issue by leonheess** [getgrav/grav#3473]
Hi everyone, I'm new to grav and inherited an old grav page (v1.0.10). I'd like to upgrade it to the newest version but my attempts have been futile.

I tried:
1. Updating via admin UI --> instantly kills page even if I try to upgrade any plugin
2. Upgrading via CLI -->

    ```
    Grav v1.7.23 is now available [release date: Wed Sep 29 20:23:09 2021].
    You are currently using v1.0.10.
    Would you like to read the changelog before proceeding? [y|N] N
    Would you like to upgrade now? [y|N] y
    
    Preparing to upgrade to v1.7.23..
      |- Downloading upgrade [8.56M]...   100%
      |- Installing upgrade...    ok
      '- Success!
    
    PHP Fatal error:  Uncaught Error: Interface 'Symfony\Contracts\EventDispatcher\EventDispatcherInterface' not found in /var/www/domino-catering.de/vendor/symfony/event-dispatcher/EventDispatcherInterface.php:23
    Stack trace:
    #0 /var/www/domino-catering.de/vendor/composer/ClassLoader.php(412): include()
    #1 /var/www/domino-catering.de/vendor/composer/ClassLoader.php(301): Composer\Autoload\includeFile()
    #2 [internal function]: Composer\Autoload\ClassLoader->loadClass()
    #3 /var/www/domino-catering.de/vendor/symfony/event-dispatcher/EventDispatcher.php(33): spl_autoload_call()
    #4 /var/www/domino-catering.de/vendor/composer/ClassLoader.php(412): include('/var/www/domino...')
    #5 /var/www/domino-catering.de/vendor/composer/ClassLoader.php(301): Composer\Autoload\includeFile()
    #6 [internal function]: Composer\Autoload\ClassLoader->loadClass()
    #7 /var/www/domino-catering.de/vendor/rockettheme/toolbox/Event/src/EventDispatcher.php(16): spl_autoload_call()
    #8 /var/www/domino-catering.de/vendor/composer/ClassLoader.php(412): include('/var/w in /var/www/domino-catering.de/vendor/symfony/event-dispatcher/EventDispatcherInterface.php on line 23
    
    ```
3. Getting the newest version to run and importing the old data --> All HTML-Code is rendered in plaintext
4. https://learn.getgrav.org/17/advanced/grav-development/grav-15-upgrade-guide --> 

    ![image](https://user-images.githubusercontent.com/13834296/137603364-4cc14099-998e-486d-8649-92027a9cffa8.png)

Can someone give me a hint?

**Comment by rhukster** [comment]
That's a very old version, and my suggestion is to simply download the upload package: https://github.com/getgrav/grav/releases/download/1.7.23/grav-update-v1.7.23.zip

Then you can just unzip and copy the files on top of your current Grav installation

**Comment by sebastianbaumann** [comment]
@leonheess 

This behavior is a result of the new default of auto-escaping being true in Grav 1.7. 

Have a look here: https://learn.getgrav.org/17/advanced/grav-development/grav-17-upgrade-guide#most-common-issues and here: https://learn.getgrav.org/17/advanced/grav-development/grav-16-upgrade-guide#twig
This should help you. 

TL;DR: you have to change your template files and make sure you use the `|raw` filter.
